### DISCUSSION TITLE
Which License is best for the open source project? MIT? / The Unlicense?

### DISCUSSION THREAD

**Discussion by Arvindjitta** [EddieHubCommunity/support#92] [Q&A] [answered]
When i was  trying to make new open source repository.....I've got stuck at selecting appropriate license....  Which License is best for the open source project? MIT? / The Unlicense?

**Comment by eddiejaoude** [comment, marked as answer]
This is a great question 👍 , not sure I know the "right" answer though 🤓 

A good place to look is http://choosealicense.com by **GitHub**

Both the `MiT` and `The Unlicesnse` seem very similar apart from the condition of `license and copyright notice`

| Feature | MIT | The Unlicense | Details
| :--- | :--- | :--- | :--- |
| Commercial use | ✅ | ✅ | The licensed material and derivatives may be used for commercial purposes |
| Distribution | ✅ | ✅ | The licensed material may be distributed |
| Modification | ✅ | ✅ | The licensed material may be modified |
| Private use | ✅ | ✅ | The licensed material may be used and modified in private |
| License and copyright notice | ✅ | - | A copy of the license and copyright notice must be included with the licensed material |
| Liability | ✅ | ✅ | This license includes a limitation of liability |
| Warranty | ✅ | ✅ | This license explicitly states that it does NOT provide any warranty |

**Comment by MikeRogers0** [comment]
I normally use `MIT` for stuff which is 100% open source, it kind of feels like the standard choice right now.

I have been using [The Prosperity Public License 3.0.0](https://licensezero.com/licenses/prosperity) in a few of my projects. It lets you be open source, while still requesting users pay you some money.

**Reply by eddiejaoude** [reply]
Thanks @MikeRogers0 , that is an interesting one, I didnt know about it, I will take a look 👍

**Comment by HarshKapadia2** [comment]
Almost all of my projects are under the MIT license, but my [git_basics](https://github.com/HarshKapadia2/git_basics) repo is under the CC-BY-SA-4.0 license as it is a documentation project.

**Reply by eddiejaoude** [reply]
Why did you use a different license for your documentation project?

**Reply by HarshKapadia2** [reply]
I was exploring licenses and I found [CC-BY-SA-4.0](https://choosealicense.com/licenses/cc-by-sa-4.0/), a license which had some more conditions than the MIT license.

I have compiled the documentation from various sources (that I have linked all over the document at the appropriate places) and I wanted any further modifications to be open sourced with appropriate credits provided to the people whom the content originally belonged to and to be released under the same license, which forces any further modifications also to have all the credits and that's a chain that I wanted to have.

I don't think the MIT license forces you to do that.

Do my points make sense? Do correct me if I am wrong. 👍

**Reply by eddiejaoude** [reply]
Ah oh, makes sense, thank you 👍

**Comment by stemount** [comment]
So this doesn't remain "unanswered" forever (as it is an open ended question @Arvindjitta - please raise another issue or add to this discussion if you have a project in mind), I have marked @eddiejaoude post as the answer with the link to https://choosealicense.com (hopefully being up to date enough).

The Prosperity licenses are interesting though as Open Source Software needs financial backing!

**Reply by eddiejaoude** [reply]
Thanks @stemount , good idea 👍
